content/blog/2021-06-04-community-spotlight/contents.lr

```diff
@@ -0,0 +1,27 @@
+title: InvenioRDM Community Spotlight: Summer 2021
+---
+author: Sara Gonzales
+---
+pub_date: 2021-06-07
+---
+body:
+
+![](Summer.jpg)
+
+Summer 2021 is an exciting time in the development of InvenioRDM, as the team works towards the Long-Term Support (LTS) version. Here are just a few updates we have to share about recent work:
+
+## Contributions from the community
+
+InvenioRDM partners are not only local implementers, but frequently contribute their coding expertise to the project. Read [this recent news item](https://www.tuwien.at/en/research/rti-support/research-data/news/news/tu-wien-contributes-to-inveniordm-development) about the efforts of TU Wien developer Maximilian Moser related to the share-by-link feature and the authentication modules.
+
+
+## Usability testing
+
+One way that the entire community can contribute towards InvenioRDM development is to test the most recent version of the software either through your local implementation or at the [CERN sandbox site](https://inveniordm.web.cern.ch/). Report any bugs you find using this [form](https://docs.google.com/forms/d/e/1FAIpQLSdExcCjDUMTkaXCU9JHCBqeFfDuCW2kFQGbuZFrr3t77qHOsw/viewform?usp=sf_link). Timely and accurate bug reporting gives the entire project a boost! 
+
+
+## Getting the word out
+
+InvenioRDM will be featured in two sessions at [Open Repositories 2021, June 7-10](https://or2021.openrepositories.org/), “Poster Minute Madness” and the “Repository Rodeo,” both taking place on June 9. If you or someone you know would like an introduction to the software, encourage them to attend!
```

Comment:
Should the comma in "Repository Rodeo," be outside? I feel like I have seen comman inside before though...
```suggestion
InvenioRDM will be featured in two sessions at [Open Repositories 2021, June 7-10](https://or2021.openrepositories.org/), “Poster Minute Madness” and the “Repository Rodeo”, both taking place on June 9. If you or someone you know would like an introduction to the software, encourage them to attend!
```

Reply:
Outside probably is more grammatically correct. :-)